.jhipster/GeocodingResult.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "GeocodingResult",
+  "fields": [
+    {
+      "fieldName": "address",
+      "fieldType": "String",
+      "fieldValidateRules": [
+        "required"
+      ]
+    },
+    {
+      "fieldName": "latitude",
+      "fieldType": "Double",
+      "fieldValidateRules": [
+        "required"
+      ]
+    },
+    {
+      "fieldName": "longitude",
+      "fieldType": "Double",
+      "fieldValidateRules": [
+        "required"
+      ]
+    }
+  ],
+  "relationships": [],
+  "changelogDate": "20190222165806",
+  "entityTableName": "geocoding_result",
+  "dto": "mapstruct",
+  "pagination": "no",
```

Comment:
please add pagination since admin won't be able to check these values when they will become a larger set